include/openmc/particle_data.h

```diff
@@ -164,6 +164,32 @@ struct BoundaryInfo {
 //! Defines how particle data is laid out in memory
 //============================================================================
 
+/*
+ * This class was added in order to separate the layout and access of particle
+ * data from particle physics operations during a development effort to get
+ * OpenMC running on GPUs. In the event-based Monte Carlo method, one creates
+ * an array of particles on which actions like cross section lookup and surface
+ * crossing are done en masse, which works best on vector computers of yore and
+ * modern GPUs. It has been shown in the below publication that arranging
```

Comment:
The comment mentions "the below publication," but I'm not seeing where the publication is listed.

Reply:
ah, good catch. Putting it in now...